src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -543,6 +543,9 @@
   - name: Set using both key-value pair and a json file with more settings.
     text: >
         az webapp config appsettings set -g MyResourceGroup -n MyUniqueApp --settings mySetting=value @moreSettings.json
+parameters:
+    - name: --settings
+      short-summary: Space-separated appsettings in a format of either <name>=<value> or @<json_file>.
```

Comment:
Angle brackets do not appear correctly in docs. Suggest the following for consistency with similar uses in the CLI:
```
Space-separated appsettings in KEY=VALUE format. Use @{file} to load from a file.
```

Reply:
Also, since you note that this command and slot-settings should match, could you update that to be consistent with the above as well?  Notice in your screenshot how the name=value only shows the equals sign? That's because of the angle brackets.

Reply:
Do you mean I should add another `parameters` for `--slot-settings`? Or do you mean I should update the `--settings` summary to use "@moreSettings.json" instead of "Use @{file}".

Reply:
Basically, copy-paste your parameters entry but for `--slot-settings`. That way they both match (as you pointed out in your screenshot), they are both consistent, and they will both appear correctly in docs.